.env.dist

```diff
@@ -1,5 +1,12 @@
 #DEFAULT_LOCALE=en
 #SERVICE_NAME=articles
+#S3_BUCKET=bucket
+#S3_PUBLIC_ENDPOINT=https://bucket.s3.us-east-1.amazonaws.com
```

Comment:
Can probably make this optional by using a compiler pass to work out it if it's not set. One for later though.